sktime/classification/kernel_based/_rocket_classifier.py

```diff
@@ -233,7 +249,8 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
-        if parameter_set == "results_comparison":
```

Comment:
we should leave the `results_comparison` parameter set here, this is used in consistency experiments.

I.e., there would be an "if", but no "else".